docs/Fachartikel/Fachartikel.tex

```diff
@@ -113,50 +122,92 @@ \subsection{Risiken}
+
+\rowcolors{2}{gray!10}{gray!40}
+\begin{table}[ht]
+\begin{tabular}{ | l | c | c |}
+	\hline
+    \textbf{Praktikum} & \textbf{Lauffähigkeit} & \textbf{Fehlende Funktion} \\
+	\hline
+    1 Target System & Nein & C Code \\
+	\hline
+    2 Bit Manipulations & Nein & C Code \\
+\hline
+    3 Introduction to Assembly & Ja* & - \\
+\hline
+    4 Data Transfer Commands & Ja & - \\
+\hline
+    5 Arithmetic Operations & Ja & - \\
+\hline
+    6 ALU and Branch Instructions & Ja & - \\
+\hline
+    7 Control Structures & Ja & - \\
+\hline
+    8 Structured Programming in Assembly & Ja & - \\
+\hline
+    9 Subroutines and Parameter Passing & Nein & Potentiometer \\
+\hline
+    10 Multiplication & Nein** & Linker \\
+\hline
+    11 Modular Coding and Linking & Nein & C Code \\
+\hline
+    12 Interrupt & Nein & Interrupts \\
+\hline
+\end{tabular}
+* Vorgegebener Code muss leicht angepasst werden.
+\newline** Wäre nach einigen Anpassungen am Programmrahmen ebenfalls lauffähig.
+\caption[size=8pt]{Lauffähigkeitsstatus aller Praktika des CT1-Moduls auf dem virtuellen CT Board.}
+\label{praktika}
+\end{table}
 
 \section{Diskussion und Ausblick}
 
-Die aktuelle Version der Software ermöglicht es, einige der Praktika des CT1-Moduls virtuell durchzuführen. Damit ist der Grundstein für die Ersetzung der physischen CT Boards gelegt. Die Studierenden können nach und nach auf das \glqq Virtual CT Board\grqq{} umsteigen - wenn auch noch nicht vollständig. Bis alle Praktika von CT1 auf dem \glqq Virtual CT Board\grqq{} lauffähig sind, könnten im CT1-Unterricht die physischen Boards neben dem virtuellen eingesetzt und damit die bestehenden Ressourcen ideal ausgeschöpft werden. Falls die ZHAW Interesse daran hat, das \glqq Virtual CT Board\grqq{} weiter zu entwickeln und auch finanziell zu unterstützen, bietet sich die Umsetzung folgender Punkte an, für die die aufgewendete Zeit im Rahmen des PM4-Moduls nicht ausgereicht hat:
+Die aktuelle Version der Software ermöglicht es, einige der Praktika des CT1-Moduls virtuell durchzuführen. Damit ist der Grundstein für die Ersetzung der physischen CT Boards gelegt. Die Studierenden können nach und nach auf das \glqq Virtual CT Board\grqq{} umsteigen - wenn auch noch nicht vollständig. Bis alle Praktika von CT1 auf dem \glqq Virtual CT Board\grqq{} lauffähig sind, könnten im CT1-Unterricht die physischen Boards neben dem virtuellen eingesetzt und damit die bestehenden Ressourcen ideal ausgeschöpft werden. Falls die ZHAW Interesse daran hat, das \glqq Virtual CT Board\grqq{} weiter zu entwickeln und auch finanziell zu unterstützen, bietet sich die Umsetzung folgender Punkte an, für die die aufgewendete Zeit im Rahmen des PM4-Moduls nicht ausgereicht hat (siehe hierzu auch Tabelle \ref{praktika}):
```

Comment:
Umsetzung folgender Punkte an, für **welche** die aufgewendete Zeit (anstelle die die)

Reply:
'welch(e/r)" gilt in einschlägigen Germanisten-Kreisen eigentlich eher als veraltet (selbst in einem Fall wie diesem). Mir kommt's aber nicht draufan :-)